spec.html

```diff
@@ -142,6 +153,11 @@ <h1>Conformance</h1>
   <p>A conforming implementation of ECMAScript may support program and regular expression syntax not described in this specification. In particular, a conforming implementation of ECMAScript may support program syntax that makes use of any &ldquo;future reserved words&rdquo; noted in subclause <emu-xref href="#sec-keywords-and-reserved-words"></emu-xref> of this specification.</p>
   <p>A conforming implementation of ECMAScript must not implement any extension that is listed as a Forbidden Extension in subclause <emu-xref href="#sec-forbidden-extensions"></emu-xref>.</p>
   <p>A conforming implementation of ECMAScript must not redefine any facilities that are not implementation-defined, implementation-approximated, or host-defined.</p>
+  <p>A conforming implementation of ECMAScript may choose to implement or not implement Normative Optional subclauses. However if Normative Optional behavior is implemented in some form, it must be implemented in accordance to the Normative Optional clause. A Normative Optional clause is defined in the spec within a Normative Optional colored box.</p>
+  <emu-clause id="sec-conformance.normative-optional" normative-optional>
```

Comment:
```suggestion
  <emu-clause id="sec-conformance.normative-optional" type="example" normative-optional>
```

@bakkot I know we use the `type="example"` attribute for `emu-grammar` elements. Should we also be using it here?

Reply:
I believe it wouldn't affect anything the current setup, but it wouldn't hurt.